common/wireaddr.c

```diff
@@ -31,7 +45,7 @@ bool fromwire_wireaddr(const u8 **cursor, size_t *max, struct wireaddr *addr)
 	return *cursor != NULL;
 }
 
-void towire_wireaddr(u8 **pptr, const struct wireaddr *addr)
+void towire_wireaddr(u8 ** pptr, const struct wireaddr *addr)
```

Comment:
And another "one-space" change. Seems like this was not done with a higher purpose. Right?